src/org/parosproxy/paros/network/HttpMethodHelper.java

```diff
@@ -59,19 +59,13 @@
 	private static final String PUT		= "PUT";
 	private static final String DELETE	= "DELETE";
 	private static final String TRACE	= "TRACE";
-	// TODO: What's up with the CONNECT method?
-	//private static final String CONNECT	= "CONNECT";
-	
 	
 	private static final String CRLF	=	"\r\n";
 	private static final String LF		=	"\n";
 	private static final Pattern patternCRLF	= Pattern.compile("\\r\\n", Pattern.MULTILINE);
 	private static final Pattern patternLF		= Pattern.compile("\\n", Pattern.MULTILINE);
 	
-	private String mUserAgent = "";
-	
 	public void setUserAgent(String userAgent) {
-		mUserAgent = userAgent;

```

Comment:
If the variable is being dropped and the assignment, why keep the method? (Just in case there's some 3rd party code we don't know about?)

If it's being left, add appropriate deprecation details?